.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Code owners for edx-ora2, open response assessment repo
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence, they will
+# be requested for review when someone opens a pull request.
+*       @edx/masters-devs-gta @mattcarter
```

Comment:
not sure about calling out an individual vs a team - i.e: remove `@mattcarter` ?

Reply:
I would like to just do a team, but right now the `edx/masters-devs-gta` can't be called as a reviewer... gotta figure that out.

Reply:
We should find alignment with `openedx.yaml`; between `supporting_teams` and `owner`  which is us and me, respectively ATM. I think the setup you currently have proposed is fine